frontend/src/pages/projects/screens/projects/ProjectTableRowNotebookTableRow.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Td, Tr } from '@patternfly/react-table';
+import { NotebookKind, ProjectKind } from '~/k8sTypes';
+import NotebookRouteLink from '~/pages/projects/notebook/NotebookRouteLink';
+import NotebookStateStatus from '~/pages/projects/notebook/NotebookStateStatus';
+import { getDisplayNameFromK8sResource } from '~/concepts/k8s/utils';
+import { NotebookState } from '~/pages/projects/notebook/types';
+import { useNotebookActionsColumn } from '~/pages/projects/notebook/NotebookActionsColumn';
+
+type ProjectTableRowNotebookTableRowProps = {
+  project: ProjectKind;
+  obj: NotebookState;
+  onNotebookDelete: (notebook: NotebookKind) => void;
+  enablePipelines: boolean;
+};
+const ProjectTableRowNotebookTableRow: React.FC<ProjectTableRowNotebookTableRowProps> = ({
+  project,
+  obj: notebookState,
+  onNotebookDelete,
+  enablePipelines,
+}) => {
+  const [ActionColumn, stopNotebook] = useNotebookActionsColumn(
+    project,
+    notebookState,
+    enablePipelines,
+    onNotebookDelete,
+  );
+  return (
+    <Tr style={{ border: 'none' }} data-testid="project-notebooks-table-row">
+      <Td dataLabel="workbenchName" style={{ paddingLeft: 'var(--pf-v5-global--spacer--sm)' }}>
```

Comment:
cell value is misaligned with header:
![image](https://github.com/user-attachments/assets/70ab5c1b-5271-40cc-bfdb-90a3b8d6cfa0)